apispec/ext/marshmallow/openapi.py

```diff
@@ -392,6 +391,45 @@ def field2property(self, field, use_refs=True, dump=True, load=True, name=None):
 
         return ret
 
+    def resolve_nested_schema(self, schema):
+        """Method to resolve a dictinary from schemas
```

Comment:
Nit: "Method to" is unnecessary. Also, typo in "dictionary. How about "Return the Open API representation of a marshmallow `Schema`."